source/features/parse-backticks.tsx

```diff
@@ -25,6 +25,10 @@ function init(): void {
 		'.Box-row.js-navigation-item a.link-gray-dark', // `isGlobalDiscussionList`
 		'[data-channel^="check_suites"] a', // `isActions`
 		'.repository-content .pr-toolbar h2', // `isActions` run
+		'.Details[data-issue-and-pr-hovercards-enabled] .Details-content--hidden a:first-child', // `isRepoRoot` ("Repository refresh" layout)
+		'.Details[data-issue-and-pr-hovercards-enabled] .Details-content--hidden pre', // `isRepoRoot` ("Repository refresh" layout)
+		'.js-issue-title', // `isIssue` ("Repository refresh" layout)
```

Comment:
The `master` version already works for me, is this required? 

Example: https://github.com/sindresorhus/refined-github/issues/2852

Reply:
It didn't work on some issues when I added it, but I think it can be removed now.